src/pomodoro/containers/PomodoroTitle.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import format from 'date-fns/format'
+import { Helmet } from 'react-helmet'
+import { connect } from 'react-redux'
+import * as fromPomodoro from 'pomodoro/store'
+
+const PomodoroTitle = ({ lapse, title }) => (
+  <Helmet>
+    <title>🕑 {format(lapse, 'mm:ss')} 🗒️ {title}</title>
+  </Helmet>
+)
+
+const mapStateToProps = state => ({
```

Comment:
Can you just move the `mapState` portion to below the propTypes declaration? Just to keep the consistency beyond the other components.